client/src/components/TeamUserListModal/index.js

```diff
@@ -62,16 +63,14 @@ const UsersInTeamModal = () => {
     ));
   };
   return (
-    <div>
-      <Button type="primary" onClick={() => setVisible(true)}>
-        Users
-      </Button>
-      <StyledModal onCancel={() => setVisible(false)} visible={visible} title="Users" footer={null}>
-        {ListOfUsers()}
-      </StyledModal>
-    </div>
+    <StyledModal onCancel={handleCancel} visible={visible} title="Users" footer={null}>
```

Comment:
Is passing null to the footer a good idea, maybe we could use some placeholder here whose value is `null` so that later we have room for improvisation.

Reply:
As we are not using footer on any of our modals, null might be a better choice

Reply:
Cool.